yt/frontends/ramses/io.py

```diff
@@ -306,6 +310,7 @@ def _read_particle_subset(self, subset, fields):
         return tr
 
 
+@lru_cache(maxsize=32)
```

Comment:
just curious, why 32 ?

Reply:
I dropped the 32 :)

Reply:
so... why 128 then ? 😅

Reply:
This is a question for the CPython mailing list ;)

Reply:
Fine, I don't really mind :)